structure_tests/licenses_test.go

```diff
@@ -0,0 +1,66 @@
+package structure_tests
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path"
+	"strings"
+	"testing"
+)
+
+type problem struct {
+	name  string
+	issue string
+}
+
+func TestImageLicenses(t *testing.T) {
+	whitelist := [0]string{}
```

Comment:
It's a bit more idiomatic to write these as slices rather than arrays:

whitelist := []string{}
blacklist := []string{"AGPL"...}